defaults/main.yaml

```diff
@@ -303,3 +303,5 @@ grafana_config:
   log.console: "{{ grafana_log__console }}"
   log.file: "{{ grafana_log__file }}"
   event_publisher: "{{ grafana_event_publisher }}"
+
+grafana_apt_repo: deb https://packagecloud.io/grafana/stable/debian/ wheezy main
```

Comment:
You define the default here but then neglect to use it in the install task. Should also probably be defined using the distro release a la 

`deb https://packagecloud.io/grafana/stable/debian/ {{ansible_distribution_release}} main`